src/collection-api/routes/docs.test.js

```diff
@@ -4,7 +4,7 @@ import request from 'supertest';
 
 import app from '../server.js';
 
-const basePath = config.get('@opentermsarchive/engine.api.basePath');
+const basePath = config.get('@opentermsarchive/engine.collection-api.basePath');
```

Comment:
Shouldn't it be in camelcase?

Reply:
`collection-api` here is the name of the module (even if this module is for now embedded in the engine), and as module name we use kebab-case and not camelCase, like `@opentermsarchive/federation-api`.